etc/profile-m-z/postman.profile

```diff
@@ -0,0 +1,22 @@
+# Firejail profile for postman
+# This file is overwritten after every install/update
+# Persistent local customizations
+include postman.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/Postman
+mkdir ${HOME}/Postman
+whitelist ${HOME}/Postman
+
+noblacklist ${HOME}/.config/Postman
+mkdir ${HOME}/.config/Postman
+whitelist ${HOME}/.config/Postman
+
+#private-opt postman
+private-bin postman,electron,electron[0-9],electron[0-9][0-9],locale,sh
+private-etc alternatives,ca-certificates,crypto-policies,fonts,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,localtime,nsswitch.conf,pki,resolv.conf,ssl
```

Comment:
We're going through a [rework of the private-etc](#5610) option at the moment. New syntax would be `private-etc @network,@tls-ca`. To keep a potential [revert](#5645) in that context as easy as possible, we better stick to the old syntax until further notice. So there's nothing for OP to do here. @kmk3 Any thoughts on how best to handle this?

Reply:
> We're going through a [rework of the
> private-etc](https://github.com/netblue30/firejail/pull/5610) option at the
> moment. New syntax would be `private-etc @network,@tls-ca`. To keep a
> potential [revert](https://github.com/netblue30/firejail/pull/5645) in that
> context as easy as possible, we better stick to the old syntax until further
> notice. So there's nothing for OP to do here. @kmk3 Any thoughts on how best
> to handle this?

Thanks for the heads up.

I would be inclined to keep being explicit about the paths.

It should be easy to convert to groups later if that's desired.

While doing the reverse is trickier, as for example removing a path from a
group (for hardening) without breaking profiles is non-trivial (since it would
be unclear which profiles actually depended on the path).

I'll try to expand on this later on #5645.